dashboards-observability/public/components/notebooks/components/paragraph_components/__tests__/__snapshots__/para_input.test.tsx.snap

```diff
@@ -274,7 +274,7 @@ exports[`<para_input /> spec renders the visualization component 1`] = `
                           data-test-subj="superDatePickerstartDatePopoverButton"
                           title="2020-07-21T18:37:44.710Z"
                         >
-                          2020-07-21T18:37:44+00:00
+                          2020-07-22T00:07:44+05:30
```

Comment:
As mentioned by Amruta earlier, try not updating dates in snapshot.

Reply:
Done. Thanks Subrat!